app/test_main.py

```diff
@@ -1 +1,56 @@
-# write your code here
+from pytest import MonkeyPatch
+
+from app.main import can_access_google_page
+
+
+class TestCanAccessGooglePage:
```

Comment:
mb we can try to use parametrize or etc.?)
then code will be DRY imho

Reply:
Yurii said that doing tests this way is better because you can run every test separately.